tsdb/cmd/tsdb/main.go

```diff
@@ -64,9 +66,13 @@ func execute() (err error) {
 		analyzeBlockID       = analyzeCmd.Arg("block id", "block to analyze (default is the last block)").String()
 		analyzeLimit         = analyzeCmd.Flag("limit", "how many items to show in each list").Default("20").Int()
 		dumpCmd              = cli.Command("dump", "dump samples from a TSDB")
+		dumpFormat           = dumpCmd.Flag("format", "output format").Default("stdout").String()
 		dumpPath             = dumpCmd.Arg("db path", "database path (default is "+defaultDBPath+")").Default(defaultDBPath).String()
 		dumpMinTime          = dumpCmd.Flag("min-time", "minimum timestamp to dump").Default(strconv.FormatInt(math.MinInt64, 10)).Int64()
 		dumpMaxTime          = dumpCmd.Flag("max-time", "maximum timestamp to dump").Default(strconv.FormatInt(math.MaxInt64, 10)).Int64()
+		dumpLabelName        = dumpCmd.Flag("label-name", "label name").String()
+		dumpLabelValue       = dumpCmd.Flag("label-value", "label value").String()
+		dumpSourceName       = dumpCmd.Flag("source-name", "path to sqlite3 db file").Default("./sqlite3.db").String()
```

Comment:
source-name is probably the wrong name, I think it should be more `dump-output` or so.

Reply:
We might support other output formats in the future so I would call it `output-file` or similar